spec/api/entities/search_result_spec.rb

```diff
@@ -66,7 +66,8 @@
           risk_based_bills: 0,
           injection_errored: 0,
           cav_warning: 0,
-          supplementary: 0
+          supplementary: 0,
+          lgfs_hardship: 0
```

Comment:
@kmahern I think you need to  add a search_result_spec example to cover this filer type - see the examples in this file - agfs equivalent should too

Reply:
👍 see also https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3282